deployment/kubernetes/generic/k8s-1-9-and-above/broker.yaml

```diff
@@ -25,17 +25,17 @@ metadata:
 data:
     # Tune for available memory. Increase the heap up to 24G to have
     # better GC behavior at high throughput
-    PULSAR_MEM: "\" -Xms64m -Xmx128m -XX:MaxDirectMemorySize=128m\""
-    zookeeperServers: zookeeper
-    configurationStoreServers: zookeeper
-    clusterName: local
+    PULSAR_PREFIX_PULSAR_MEM: "\" -Xms64m -Xmx128m -XX:MaxDirectMemorySize=128m\""
+    PULSAR_PREFIX_zookeeperServers: zookeeper
+    PULSAR_PREFIX_configurationStoreServers: zookeeper
+    PULSAR_PREFIX_clusterName: local
     # change the managed ledger settings if you have more bookies
-    managedLedgerDefaultEnsembleSize: "1"
-    managedLedgerDefaultWriteQuorum: "1"
-    managedLedgerDefaultAckQuorum: "1"
+    PULSAR_PREFIX_managedLedgerDefaultEnsembleSize: "1"
+    PULSAR_PREFIX_managedLedgerDefaultWriteQuorum: "1"
+    PULSAR_PREFIX_managedLedgerDefaultAckQuorum: "1"
     # enable pulsar functions
-    functionsWorkerEnabled: "true"
-    PF_pulsarFunctionsCluster: local
```

Comment:
There was already a prefix for pulsar functions: 
```PF_ENV_PREFIX = 'PF_'```
and `gen-yml-from-env.py` will handle it. we may not need to handle it.

Reply:
Yes, will fix it